test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -223,36 +195,41 @@ describe('Error related test cases,', function() {
 
         it('Error modal text indicates users about error and provides them with navigation options to Home Page', function(){
             var modalText = chaisePage.recordPage.getModalText();
-            expect(modalText.getText()).toBe(testParams.sizeNotValidModalText(), "The message in modal pop is not correct");
+            expect(modalText.getText()).toBe(testParams.negativeLimitErrorText, "The message in modal pop is not correct");
         });
 
-        it('On click of OK button the page should redirect to RecordSet', function(){
+        it('On click of OK button the page should redirect to Home page', function(done){
             chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
                 return btn.click();
             }).then (function (){
                 return browser.driver.getCurrentUrl();
             }).then (function(currentUrl) {
-              var newapplink = url.replace("recordedit", "recordset"),
-                  lastSlash = newapplink.lastIndexOf("/"),
-                  recordsetUrl = newapplink.slice(0, lastSlash);
-                expect(currentUrl).toBe(recordsetUrl, "The redirection from recordedit page to recordset failed");
+              var homeAppUrl = browser.params.url,
+                  homePage =   homeAppUrl.slice(0, homeAppUrl.slice(0, homeAppUrl.lastIndexOf("/")).lastIndexOf("/") + 1);
```

Comment:
This is a little confusing as to what you are doing here. Please add a comment as to what the url you are working with looks like and what it's being changed to for the comparison.